podvm-mkosi/mkosi.presets/system/mkosi.conf.d/fedora-debug.conf

```diff
@@ -0,0 +1,16 @@
+[Match]
+Distribution=fedora
+# This is a little hack to define different image types in a mkosi config.
+# There is also imageId, but it renames the output, which is not what we want.
+PathExists=../../resources/buildDebugImage
+
+[Content]
+Autologin=true
+KernelCommandLine=rd.shell
+KernelCommandLine=systemd.setenv=SYSTEMD_SULOGIN_FORCE=1
+Packages=
+    nano
+    vim
+    strace
+    openssh-server
```

Comment:
if we have openssh we probably want sudo, or can the key be provisioned for root?

Reply:
I think on the current image we'd also need a passwordless sudoers directive:

```
cat etc/sudoers.d/wheel-no-passwd
%wheel  ALL=(ALL)       NOPASSWD: ALL
```

Reply:
SSH is now working as expected for root and the key embedded in the debug image. I've decided to ignore the cloud init key, will work on throwing that out next.